packages/backend/src/telemetry/privacy.test.ts

```diff
@@ -0,0 +1,95 @@
+import { privacy } from './privacy'
+describe('Privacy functions', () => {
+  const clipParams = {
+    minBucketSize: 1000,
+    maxBucketSize: 10000
+  }
+
+  afterEach(() => {
+    jest.unmock('./privacy')
+  })
+
+  test('generateLaplaceNoise should return a different number each time', () => {
```

Comment:
I think in this test we should just mock `Math.random` and verify the deterministic parts

Reply:
i added another test for your suggestion, have a look.

Also:
would you want me to remove this one entirely? I'm presuming you are thinking that the test might fail every now and then since theoretically Math.random is pseudo-random.
I'd argue that we won't hit it :D